### PR TITLE
iCarousel: Disable modules.

### PR DESCRIPTION
**Pull request by Shachlan**

### CODE REVIEW

File: iCarousel.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -404,6 +426,7 @@
 				821507491E57475B00C75711 /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
```
Comment (barakwei) [inline iCarousel.xcodeproj/project.pbxproj]: Can you remove this please?

File: iCarousel.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -17,13 +20,20 @@
...
 /* End PBXFileReference section */
 
 /* Begin PBXFrameworksBuildPhase section */
 		821507271E57468200C75711 /* Frameworks */ = {
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				C73646A62033357200C47732 /* CoreGraphics.framework in Frameworks */,
+				C73646A42033355100C47732 /* QuartzCore.framework in Frameworks */,
+				C73646A02033350000C47732 /* UIKit.framework in Frameworks */,
```
Comment (barakwei) [inline iCarousel.xcodeproj/project.pbxproj:35]: This is the real list that you link with. `CoreMedia` is not here, but it is in other places in the PR. Do we need it or not?

File: iCarousel.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Shachlan) [reply iCarousel.xcodeproj/project.pbxproj]: done.

File: iCarousel.xcodeproj/project.pbxproj (hunk 2 above)
Comment (Shachlan) [reply iCarousel.xcodeproj/project.pbxproj:35]: not needed. For some reason it wasn't removed from the whole project when I removed it.

**Comment by Shachlan** [comment]
Finished squashing.

**Review by barakwei — APPROVED** [review]

**Pull request merged**